packages/styles/src/components/_picker-list.scss

```diff
@@ -0,0 +1,34 @@
+$base-class: 'lc-picker-list';
+
+.#{$base-class} {
+  background-color: var(--surface-basic-default);
+  border: 1px solid var(--border-default);
+  border-radius: 4px;
+  bottom: -231px;
+  box-sizing: border-box;
+  font-size: 15px;
+  font-weight: 400;
+  height: 224px;
+  line-height: 22px;
+  overflow: scroll;
+  padding: 4px 0;
+  position: absolute;
+  width: 341px;
+
+  &_item {
```

Comment:
If we follow the BEM (and I think we do) then we should use double underscore `__`